Formula/jless.rb

```diff
@@ -1,12 +1,11 @@
 # jless (Jam Less) is Japan-ized Less.
 # jless supports ISO 2022 code extension techniques and Japanese codes.
 class Jless < Formula
-  desc "Jless file pager supporting ISO2022"
+  desc "File pager supporting ISO2022"
   homepage "http://www.greenwoodsoftware.com/less/"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/j/jless/jless_382-iso262.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/j/jless/jless_382-iso262.orig.tar.gz"
-  version "382+iso262+ext03"
-  sha256 "d33cc51f220647d015fd526b3782cde03c5d8b18c5233e6388a0047a70446fe7"
+  url "http://www.greenwoodsoftware.com/less/less-481.tar.gz"
+  sha256 "3fa38f2cf5e9e040bb44fffaa6c76a84506e379e47f5a04686ab78102090dda5"
+  version_scheme 1
```

Comment:
@MikeMcQuaid Sorry, I accidentally deleted your comment when I was trying to delete my own (I started a review instead of adding a single comment). To answer the question

> Just to double-check: presumably upgrade fails without this?

No, upgrade works without this, but `382+iso262+ext03` to `481` feels like a version scheme change to me, in principle. I don't mind removing it.